api/src/wfl/service/rawls.clj

```diff
@@ -23,7 +24,8 @@
 
 (defn create-snapshot-reference
   "Link a Terra Data Repo snapshot with id SNAPSHOT-ID to a fully-qualified
-  Terra WORKSPACE as NAME, optionally described by DESCRIPTION."
+  Terra WORKSPACE as NAME (with the snapshot name as its base if unspecified),
+  optionally described by DESCRIPTION."
   ([workspace snapshot-id name description]
    (-> (workspace-api-url workspace "snapshots")
        (http/post {:headers      (auth/get-auth-header)
```

Comment:
>Not sure what (with the snapshot name as its base if unspecified) means.
Does its refer to the snapshot, the name? What might be unspecified?

Good call out, I see how this is unclear.  When creating a snapshot reference, we may specify the reference's name. If unspecified, we generate a unique reference name with the snapshot's name as its base.  Does that clarify?

>This is the rawls namespace, so can we assume the TDR here?
(Not actually sure, but I think so. Are there other varieties of snapshot?)

Yes, I think so too.  I was skittish about distinguishing between snapshots (TDR) and snapshot references (workspace) because the Rawls Swagger docs do not do a great job of distinguishing between them IMO.

Reply:
What's there is good enough.